src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    final int STACK_CAPACITY = 12;
+    final int EMPTY_STACK_VALUE = -1;
+    final int LAST_INDEX =  STACK_CAPACITY - 1;
+    private final int[] numbers = new int[STACK_CAPACITY];
+
+
+    private int total = EMPTY_STACK_VALUE;
+
+    public int getTotal() {return total;}
+
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    public boolean callCheck() {
+        return total == EMPTY_STACK_VALUE;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods.